templates/new_service/.gitignore

```diff
@@ -4,3 +4,5 @@ terraform/domains/environment_domains/vendor
 terraform.tfstate*
 bin/terrafile
 bin/konduit.sh
+
```

Comment:
extra space?

Reply:
The empty line is there on purpose to create different sections, but it may not have been clear. Created more sections with explicit headings now.